mcda/utils/utils_for_main.py

```diff
@@ -47,36 +47,41 @@ def check_config_error(condition: bool, error_message: str):
 
 def check_config_setting(condition_robustness_on_weights: bool,
                          condition_robustness_on_indicators: bool,
-                         mc_runs: int) -> (int, int):
+                         mc_runs: int, random_seed: int) -> (int, int):
     """
-        Checks configuration settings and logs information messages.
+    Checks configuration settings and logs information messages.
 
-        Parameters:
-        - condition (bool): The condition to check.
+    Parameters:
```

Comment:
List of parameters do not match to that of declared in the method definition. 
I think the parameters added here are the expected return parameters and not the ones passed to it. Can you please write both the ones passed and the ones returned in `Returns` ?

Reply:
I have corrected "Parameters" into "Returns". However, I am not adding "Parameters" because my thinking throughout the code has been to add a description of the parameters and/or returns only when I feel it needs a short explanation. Anyways, the last section in every docstring always lists all `:param:` and `:return:`.